seqr/utils/file_utils.py

```diff
@@ -12,12 +12,12 @@ def run_command(command, user=None, pipe_errors=False):
     return subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE if pipe_errors else subprocess.STDOUT, shell=True) # nosec
 
 
-def _run_gsutil_command(command, gs_path, gunzip=False, user=None, pipe_errors=False):
+def _run_gsutil_command(command, gs_path, gunzip=False, user=None, pipe_errors=False, no_project=False):
     if not is_google_bucket_file_path(gs_path):
         raise Exception('A Google Storage path is expected.')
 
     #  Anvil buckets are requester-pays and we bill them to the anvil project
-    google_project = get_google_project(gs_path)
+    google_project = get_google_project(gs_path) if not no_project else None
```

Comment:
why aren't we including a billing project when importing the gregor metadata file?

Reply:
because that workspace is a special snowflake somehow and including the billing project causes it to fail *shrug*